epub33/core/index.html

```diff
@@ -4788,15 +4791,19 @@ <h4>The <code>manifest</code> element</h4>
 
 					<p id="confreq-rendition-manifest">[=EPUB creators=] MUST list all publication resources in the
 							<code>manifest</code>, regardless of whether they are [=container resources=] or [=remote
-						resources=]. Moreover, the <code>manifest</code> MUST only list publication resources.</p>
+						resources=].</p>
 
 					<p>Note that the <code>manifest</code> is not self-referencing: EPUB creators MUST NOT specify an
 							<code>item</code> element that refers to the [=package document=] itself.</p>
 
 					<div class="note">
-						<p>Failure to provide a complete manifest of resources may lead to rendering issues. [=Reading
-							systems=] might not unzip such resources or could prevent access to them for security
-							reasons.</p>
+						<p>The manifest is only for listing publication resources. [=Linked resources=] and <a
+								href="#sec-container-file-and-dir-structure">the special files for processing the OCF
+								Container</a> are restricted from inclusion.</p>
```

Comment:
I would maybe precise:

s/the special files for processing the OCF Container/the special files for processing the OCF Container (i.e. files in the `META-INF` directory, and the `mimetype` file)/

for extra clarity.